src/scss/06_components/paragraphs/_react-search.scss

```diff
@@ -102,3 +102,72 @@ fieldset .hdbt-search--react__checkbox:not(:first-of-type) {
     }
   }
 }
+
+.component--hdbt-search--react .component__container {
```

Comment:
These styles that are using` .component--hdbt-search--react` should rather use `component--hdbt-search--ploughing-schedule` class because Ploughing search is different from other searches in a sense it's added to basic page and not landingpage and it's design is also different. If we are going to refactor other searches to use class ` .component--hdbt-search--react` in the future the other searches styles will brake because of the changes in this PR.

Reply:
Changed.